SWLOR.Game.Server/Feature/AbilityDefinition/Devices/GasBombAbilityDefinition.cs

```diff
@@ -16,9 +16,15 @@ public class GasBombAbilityDefinition : ExplosiveBaseAbilityDefinition
 
         private static void ApplyEffect(uint creature, int dmg)
         {
+            var activator = GetAreaOfEffectCreator(OBJECT_SELF);
```

Comment:
We may need to store these values as local variables. If the effect creator logs out of the server we'll go back to zero for the values.

I guess another option is to simply kill off the AOE when they log out? Not sure.